client/my-sites/themes/sheet.jsx

```diff
@@ -51,6 +53,14 @@ const ThemeSheet = React.createClass( {
 		this.props.dispatch( signup( this.props ) );
 	},
 
+	onDownloadButtonClick() {
+		if ( this.props.download ) {
```

Comment:
Ok so we can start there, and we'll review later once we have a better API-driven way to determine this?